src/Analyzers/CSharp/CodeFixes/UseParameterNullChecking/CSharpUseParameterNullCheckingCodeFixProvider.cs

```diff
@@ -47,6 +48,9 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             Document document, ImmutableArray<Diagnostic> diagnostics,
             SyntaxEditor editor, CancellationToken cancellationToken)
         {
+            // Tracking parameters which have already been fixed by a fix-all operation.
+            // This avoids crashing the fixer when the same parameter is null-tested multiple times.
+            using var _ = PooledHashSet<Location>.GetInstance(out var fixedParameterLocations);
```

Comment:
Do we need to `fixedParameterLocations.Free()` or similar at the end?

Reply:
I think `using var` handles freeing for us.